nettrace/doc.go

```diff
@@ -0,0 +1,22 @@
+// Package nettrace implement a net.Dial and net.Conn that trace connections
+// reads, ands writes. An example of hooking this into the http Transport is below
```

Comment:
Minor typo here. "ands" should be "and". Also, based on the sentence at the bottom, I am guessing it's missing a comma.

"that trace connections reads, ands writes" -> "that trace connections, reads, and writes"